addons/web/static/src/views/list/list_renderer.js

```diff
@@ -405,7 +405,10 @@ export class ListRenderer extends Component {
 
     focus(el) {
         el.focus();
-        if (["INPUT", "TEXTAREA"].includes(el.tagName)) {
+        if (
```

Comment:
Also... :-D
![image](https://user-images.githubusercontent.com/1159815/204291958-a4541abf-9d5e-40cb-b71c-84e36bc4be58.png)

Reply:
Hmmm I did not think it through, you're right, I do not see a way to test this, sorry I just wanted to use this meme so bad ^^